bodhi/server/models.py

```diff
@@ -2046,6 +2047,12 @@ def update_test_gating_status(self):
                 decision.get('unsatisfied_requirements', []))
         self.greenwave_summary_string = decision['summary']
 
+        # Comment test_gating_status change on update
+        if old_test_gating_status != self.test_gating_status:
+            self.comment(db, u"This update test_gating_status has been changed to '%s'." % (
```

Comment:
Let's make it a bit more human friendly, and here's an example of using the ```Session```:

```suggestion
            self.comment(Session(), u"This update test gating status has been changed to '%s'." % (
```